### PR TITLE
History route

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by FrancoisSalmon — DISMISSED** [review]
ok

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

**Comment by LoicBrassart** [comment]
Conflicts

**Review by RomainLF — APPROVED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by FrancoisSalmon — APPROVED** [review]
this PR was already approved by 3 people. But Aline closed it by mistake. There are no other changes.

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Comment by NtGZeko** [comment]
edit by github

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: package.json [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-    "setup": "npm i && husky install && npm --prefix ./frontend i && npm --prefix ./backend i && npm --prefix ./admin i",
-    "dev": "concurrently -n front,back -c green,yellow -t \"HH:mm:ss\" -p \"{name} {time}\" \"npm --prefix ./frontend run dev\" \"npm --prefix ./backend run dev\" \"npm --prefix ./admin run dev\"",
+    "setup": "npm i && husky install && npm --prefix ./frontend i && npm --prefix ./backend i",
+    "dev": "concurrently -n front,back -c green,yellow -t \"HH:mm:ss\" -p \"{name} {time}\" \"npm --prefix ./frontend run dev\" \"npm --prefix ./backend run dev\"",
     "dev-front": "npm --prefix ./frontend run dev",
     "dev-back": "npm --prefix ./backend run dev",
-    "lint": "npm --prefix ./frontend run lint && npm --prefix ./backend run lint && npm --prefix ./admin run lint",
-    "fix": "npm --prefix ./frontend run fix && npm --prefix ./backend run fix && npm --prefix ./admin run fix"
+    "lint": "npm --prefix ./frontend run lint && npm --prefix ./backend run lint",
+    "fix": "npm --prefix ./frontend run fix && npm --prefix ./backend run fix"
```
Comment (LoicBrassart) [inline package.json:12]: Remember when I told you not to use `git add .` ? :unamused:

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**